patches/third_party-widevine-cdm-widevine.gni.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/third_party/widevine/cdm/widevine.gni b/third_party/widevine/cdm/widevine.gni
+index e10d9b9b8d2975a731adfee2eb0086afd7975f97..3e53953d4f00f292314cc4aab0b764024bd8bc93 100644
+--- a/third_party/widevine/cdm/widevine.gni
++++ b/third_party/widevine/cdm/widevine.gni
+@@ -35,6 +35,11 @@ enable_widevine_cdm_component =
+ 
+ # Widevine CDM is bundled as part of Google Chrome builds.
+ bundle_widevine_cdm = enable_library_widevine_cdm && is_chrome_branded
++if (brave_chromium_build) {
++  # On brave linux, widevine is supported by bundle for now.
++  # TODO(simonhong): Support by component update.
```

Comment:
Mind placing a follow-up issue number here?

Reply:
has this been vetted by @bbondy? We're specifically prohibited from bundling on other platforms

Reply:
Yes, I discussed in dm and he agreed to do it as a next step.